src/Documentation/sidebar.json

```diff
@@ -4,204 +4,561 @@
+        "folder": "/static/docs/get-started",
+        "indexFile": "compare-experiments.md",
+        "files": []
+      },
+      {
+        "name": "Get Older Files",
+        "folder": "/static/docs/get-started",
+        "indexFile": "older-versions.md",
+        "files": []
+      },
+      {
+        "name": "Example: Versioning",
+        "folder": "/static/docs/get-started",
+        "indexFile": "example-versioning.md",
+        "files": []
+      },
+      {
+        "name": "Example: Pipelines",
+        "folder": "/static/docs/get-started",
+        "indexFile": "example-pipeline.md",
+        "files": []
+      }
+    ]
   },
   {
     "name": "Use Cases",
     "folder": "/static/docs/use-cases",
     "indexFile": "index.md",
     "files": [
-      "data-and-model-files-versioning.md",
-      "share-data-and-model-files.md",
-      "multiple-data-scientists-on-a-single-machine.md"
-    ],
-    "labels": {
-      "data-and-model-files-versioning.md": "Data & Model Files Versioning",
-      "share-data-and-model-files.md": "Share Data & Model Files",
-      "multiple-data-scientists-on-a-single-machine.md": "Shared Development Machine"
-    }
+      {
+        "name": "Data & Model Files Versioning",
+        "folder": "/static/docs/use-cases",
+        "indexFile": "data-and-model-files-versioning.md",
+        "files": []
+      },
+      {
+        "name": "Share Data & Model Files",
+        "folder": "/static/docs/use-cases",
+        "indexFile": "share-data-and-model-files.md",
+        "files": []
+      },
+      {
+        "name": "Shared Development Machine",
+        "folder": "/static/docs/use-cases",
+        "indexFile": "multiple-data-scientists-on-a-single-machine.md",
+        "files": []
+      }
+    ]
   },
   {
     "name": "User Guide",
     "folder": "/static/docs/user-guide",
     "files": [
-      "dvc-files-and-directories.md",
-      "dvc-file-format.md",
-      "cache-file-linking.md",
-      "external-dependencies.md",
-      "external-outputs.md",
-      "development.md",
-      "contributing.md",
-      "contributing-documentation.md",
-      "update-tracked-file.md",
-      "autocomplete.md",
-      "plugins.md",
-      "analytics.md",
-      "dvcignore.md"
-    ],
-    "labels": {
-      "dvc-files-and-directories.md": "Files and Directories",
-      "dvc-file-format.md": "File Format (.dvc)",
-      "cache-file-linking.md": "Large File Optimization",
-      "development.md": "Development Version",
-      "external-dependencies.md": "External Dependencies",
-      "external-outputs.md": "External Outputs",
-      "contributing.md": "Contributing",
-      "contributing-documentation.md": "Contributing Documentation",
-      "update-tracked-file.md": "Update a Tracked File",
-      "autocomplete.md": "Shell Autocomplete",
-      "plugins.md": "Plugins & Integrations",
-      "analytics.md": "Anonymized Usage Analytics"
-    }
+      {
+        "name" : "Files and Directories",
```

Comment:
I still don't quite understand why do we need that drastic change to the sidebar format. It's now harder to read, it has a lot of unnecessary duplication.

Reply:
sidebar.json contains the names of the destination files (indexFile), file paths, the list of attached files, and the display name. The URL directly depends on the indexFile field, so duplication of file names is unacceptable, for the reason that SideBar highlights the current file only by its name, ignoring the path. If we select a file that has a "clone" even in another directory, then both files will be highlighted, but the required one will be loaded.